src/Accounts/Accounts/Token/GetAzureRmAccessToken.cs

```diff
@@ -73,7 +71,7 @@ public class GetAzureRmAccessTokenCommand : AzureRMCmdlet
         [Parameter(Mandatory = false, HelpMessage = "Optional Tenant Id. Use tenant id of default context if not specified.")]
         public string TenantId { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Specify to convert output token as a secure string.")]
+        [Parameter(Mandatory = false, HelpMessage = "The parameter is no long used. No matter `AsSecureString` is specified, the output token is a `SecureString`.")]
```

Comment:
Making our intention clear makes it easier for the users.
```suggestion
        [Parameter(Mandatory = false, HelpMessage = "The parameter is no long used but kept for backward compatibility.")]
```